adoptingerlang.org

```diff
@@ -1763,11 +1763,13 @@ CLOSED: [2019-09-18 Wed 09:02]
 :EXPORT_HUGO_MENU: :menu main :parent production
 :END:
 
-In [[/docs/development/otp_high_level/][OTP at a High Level]] we took a bird's-eye view of how applications are combined into a release and in the follow chapters of [[/docs/development/][Development]] we built and tested OTP applications. Now we'll configure and build a release of the [[https://github.com/adoptingerlang/service_discovery][service_discovery]] project.
+In [[/docs/development/otp_high_level/][OTP at a High Level]] we took a bird's-eye view of how applications are combined into a release and in the follow chapters of [[/docs/development/][Development]] we built and tested OTP applications. We have discussed how an Erlang system is setup differently from many other languages which have a single entry point in your code, like a =main()= function that is called when starting the program. Ultimately starting an Erlang node that runs your code is similar enough to other languages and runtimes, and can be packaged up in a Docker container to run like any other container, and does not appear so different. But getting to the point that you have a bundled up release with an easy to use shell script, or a Docker image ready to run, can be confusing at first and give the impression Erlang is too different or too difficult to be running in your environment.
+
+This chapter will give some low level details about how a release is constructed, but the main purpose is to use the [[https://github.com/adoptingerlang/service_discovery][service_discovery]] project to show how to use =rebar3= for building releases suited for local development, then a production ready release and finally how to use the tools a release built with =rebar3= comes with to configure and operate a release. And ultimately show that the advantages from the flexibility of booting concurrently running applications do not have to come with the cost of esoteric operations.
```

Comment:
"then a production release, and finally"  comma before 'and'

Reply:
we use 'finally' and then 'ultimately' in the sentence after. We need to change some wording. Maybe "We will show that the..."